layouts/partials/copilot/sidebar.html

```diff
@@ -0,0 +1,5 @@
+{{ $copilotApiUrl := getenv "PULUMI_COPILOT_URL" }}
+{{ if $copilotApiUrl }}
+<aside id="ai-sidebar-target"></aside>
+<iframe id="ai-sidebar-host" src="/_pulumi/cloud-ai/sidebar?hostApp=docs&apiHost={{ $copilotApiUrl | urlquery }}"></iframe>
```

Comment:
just curious here, do we still want to show these clickable example prompts in the docs section of the site since they all seem cloud console specific to querying things about the org? I imagine this would need to be handled elsewhere though and not in this PR.

<img width="413" alt="Screen Shot 2024-09-26 at 5 42 48 PM" src="https://github.com/user-attachments/assets/22cf7fb9-b569-4482-959f-6085ffcfaeb1">

Reply:
Good one! Will make a follow up.